org.eclipse.scanning.event/src/org/eclipse/scanning/event/remote/_QueueControllerService.java

```diff
@@ -176,15 +176,15 @@ public void stopQueueService(boolean force) throws EventException {
 			String beanID = bean.getUniqueId();
 			beanState = getBeanStatus(beanID, queueID);
 		} catch (EventException evEx) {
-			logger.error("Could not get state of bean in queue. Is it in queue '"+queueID+"'? "+evEx.getMessage());
+			logger.error("Could not get state of "+bean.getClass().getSimpleName()+" '"+bean.getName()+"' in queue. Is it in queue '"+queueID+"'? "+evEx.getMessage());
 			throw evEx;
 		}
 		if (beanState.isResumed() || beanState.isRunning()) {
-			logger.error("Bean '"+bean.getName()+"' is already resumed/running.");
+			logger.error(bean.getClass().getSimpleName()+" '"+bean.getName()+"' is already resumed/running.");
 			throw new IllegalStateException("Bean '"+bean.getName()+"' is already resumed/running");
 		} else if (beanState == Status.SUBMITTED) {
-			logger.error("Bean is submitted but not being processed. Cannot resume.");
-			throw new IllegalStateException("Cannot resume a bean with SUBMITTED status");
+			logger.error(bean.getClass().getSimpleName()+" '"+bean.getName()+"' is submitted but not being processed. Cannot resume.");
```

Comment:
The state testing in QueueControllerService.java and _QueueControllerService looks like repeated code. Is it possible to somehow do this in one place.